src/index.js

```diff
@@ -1,7 +1,7 @@
 /* eslint no-unused-vars: 0 */
 import utils from './utils';
 import defaultOptions from './defaultOptions';
-import MapTable from './MapTable';
+import MapTable from 'MapTable';
```

Comment:
Not sure that this would work.
`import MapTable from './MapTable';` refers to importing the fiel `./MapTable.js`